### PR TITLE
Program was successfully updated

### PR DESCRIPTION
**Pull request by kamolbekb**

### CODE REVIEW

File: TeamChek/TeamChek/Car.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿using System;
+namespace Hometask
+{
+	public class Car
```
Comment (ShokhbozKh) [inline TeamChek/TeamChek/Car.cs:4]: kod yaxshi men tushuna oldim nima qilganizni lekin bitta car class olib ushandan boshqa moshina turlariga voris olingan joyini kurmadim umumiy xususyatlarni vor olib kodlarni kamaytiring

File: TeamChek/TeamChek/Car.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	{
+		public int Id { get; set; }
+		public string Model { get; set; }
+		public string Brand { get; set; }
+		public decimal Price { get; set; }
+		public static decimal TaxPercentage { get; set; } = 10;
```
Comment (ShokhbozKh) [inline TeamChek/TeamChek/Car.cs:10]: comment orqani 10 ga izoh bering keting

File: TeamChek/TeamChek/Car.cs (hunk 1 above)
Comment (kamolbekb) [reply TeamChek/TeamChek/Car.cs:4]: bu narsa masalaning shartida so`ralmagani uchun unga vaqt ketkazib otirmaslikka qaror qildim, KISS prinsiplesiga amal qilgan holda

**Comment by kamolbekb** [comment]
Program corresponds to the all requirements of costumer.
It has a Static value TaxPercentage, Method CalculatePrice which calculates overall price of the car based on TaxesPercentage, If costumer changes taxes percentage of a class all Tax Percentage of all the Variables will be changed automatically, and the methods will calculate based on the changed value of TaxPecentage)

File: TeamChek/TeamChek/Car.cs (hunk 1 above)
Comment (ShokhbozKh) [reply TeamChek/TeamChek/Car.cs:4]: ha unda yaxshi

**Pull request merged**